src/context/AppProvider/hooks/useAccountsFetchQuery.tsx

```diff
@@ -203,7 +203,7 @@ export const useAccountsFetchQuery = () => {
         isSnapInstalled,
       },
     ],
-    queryFn: deviceId && !hasManagedAccounts ? queryFn : skipToken,
+    queryFn: wallet && deviceId && !hasManagedAccounts ? queryFn : skipToken,
```

Comment:
The actual fix

Reply:
Nice catch ser. For anyone reading this, this is the offender above:
https://github.com/shapeshift/web/blob/db280386bf6fc1280123a12cbb76039e20f1cf3c/src/context/AppProvider/hooks/useAccountsFetchQuery.tsx#L68

While all is nice and working in an ideal world where `wallet` is defined from the first run of this, `wallet` being nullish means we will early return, which is a very valid query return value, and the query would then never re-run since we don't have `wallet` as a dep (which we can't because something something unserializable).

Ideally we would have gone with throwing instead of early returning for `!wallet`, but this is actually even better, since this will ensure a re-run, without adding non-serializable deps to the `queryKey` 🧠